src/App.css

```diff
@@ -1,12 +1,82 @@
 body {
   margin: 0;
   padding: 0;
-  font-size: 25px;
+  font-family: 'Manrope', sans-serif;
+  font-size: 18px;
+  line-height: 28px;
+  height: 100vh;
+  background-image: linear-gradient(-20deg, #2b5876 0%, #4e4376 100%);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
```

Comment:
- [ ] The background image has just the height of the viewport height. When the page length extends over the viewport height the background breaks after the viewport height and it doesn't give a good UI experience. Kindly fix this issue. :pray: 
![image](https://user-images.githubusercontent.com/64894927/139957611-c99ea3d6-3dd0-4dd9-846b-5d057ab7b569.png)